web/src/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { Provider } from 'react-redux'
+
+import { store } from './ts/store'
+import TsApp from './ts/TsApp'
```

Comment:
Also, read about react aliases
Like https://habr.com/ru/post/557076/
You will be able write just `store/user...` instead `../../store/user...`